textattack/datasets/translation/translation_datasets.py

```diff
@@ -11,5 +11,7 @@ class NewsTest2013EnglishToGerman(TextAttackDataset):
     
     """
     DATA_PATH = 'datasets/translation/NewsTest2013EnglishToGerman'
-    def __init__(self, offset=0):
+    def __init__(self, offset=0, shuffle=False):
```

Comment:
please add `shuffle` and `offset` to docstring

Reply:
done